lib/core/src/Cardano/Pool/DB/Sqlite/TH.hs

```diff
@@ -55,7 +55,7 @@ PoolProduction sql=pool_production
 -- Stake distribution for each stake pool
 StakeDistribution sql=stake_distribution
     stakeDistributionPoolId     W.PoolId     sql=pool_id
-    stakeDistributionEpoch      Word64       sql=epoch
+    stakeDistributionEpoch      Word32       sql=epoch
```

Comment:
Unfortunately, this now requires a database migration to not break every existing wallets out there. Keeping it as a `Word64` also works for now with a `TODO` tag.

Reply:
> Unfortunately, this now requires a database migration to not break every existing wallets out there. Keeping it as a `Word64` also works for now with a `TODO` tag.

Fair enough. I've reverted this to `Word64`.